app/Http/Controllers/Api/v1/TakoController.php

```diff
@@ -8,8 +8,8 @@
 
 class TakoController extends Controller
 {
-    public function filelist(){
-        $list = GamesFile::with('gamefiletype', 'game')->get();//->take(5);
+    public function filelist() {
```

Comment:
__[code_sniffer]__
* Opening brace should be on a new line

<!-- Comment by [SideCI](https://sideci.com) -->